### DISCUSSION TITLE
Set ScrollbarSize of ScrollArea used with Navbar.Section

### DISCUSSION THREAD

**Discussion by erwinstone** [mantinedev/mantine#3032] [Q&A] [answered]
How to set `ScrollbarSize` of `ScrollArea` used with `Navbar.Section`

`<Navbar.Section grow component={ScrollArea}></Navbar.Section>`

Reference: https://mantine.dev/core/scroll-area/#usage-with-other-components

**Comment by rtivital** [comment, marked as answer]
You can create custom component that will customize ScrollArea – https://codesandbox.io/s/wispy-pond-km2cz7?file=/src/App.tsx

**Reply by erwinstone** [reply]
Awesome, Thank you very much
